workers/python/kitchen_sink.py

```diff
@@ -168,10 +214,44 @@ def launch_activity(execute_activity: ExecuteActivityAction) -> Awaitable:
                 execute_activity.remote.cancellation_type
             ),
         )
-    # TODO: Handle cancels
+
     return activity_task
 
 
+async def handle_awaitable_choice(
+    awaitable: Union[Coroutine | asyncio.Task], choice: AwaitableChoice
```

Comment:
```suggestion
    awaitable: Coroutine | asyncio.Task, choice: AwaitableChoice
```

or

```suggestion
    awaitable: Union[Coroutine, asyncio.Task], choice: AwaitableChoice
```

Not both. Though really, I think `Awaitable[Any]` is best